### ISSUE TITLE
[Package Request] - hdf5-devel

### ISSUE THREAD

**Issue by hs-vc** [amazonlinux/amazon-linux-2023#296]
**What package is missing from Amazon Linux 2022? Please describe and include package name.**
hdf5-devel - development package for the HDF5 library, which is a data model, library, and file format for storing and managing large and complex data collections

**Is this an update to existing package or new package request?**
new

**Is this package available in Amazon Linux 2? If it is available via external sources such as EPEL, please specify.**
It is avaialbe in  Amazon Linux 2 via EPEL

**Any additional information you'd like to include. (use-cases, etc)**

**Comment by dejlek** [comment]
Any update on this?

**Comment by HarryCaveMan** [comment]
It's been over 6 months and not a single update on this. hdf5-devel is a major dependency for most python data science and machine learning packages. Currently building from source on amazonlinux 2023 which is definitely not ideal...

**Comment by ozbenh** [comment]
It is also a non-trivial CVE magnet. How stable is it ABI-wise in general ? Is it something that can realistically just be bumped to the latest version regularly or not ?

**Comment by dejlek** [comment]
It is actually superstable.

**Comment by HarryCaveMan** [comment]
It's also been available on amazonlinux 2 for ages

**Comment by dejlek** [comment]
>  It's also been available on amazonlinux 2 for ages

I do not think so (this is on an AL2 machine):
```
repoquery -i hdf5-devel

Name        : hdf5-devel
Version     : 1.8.12
Release     : 13.el7
Architecture: x86_64
Size        : 16175378
Packager    : Fedora Project
Group       : Development/Libraries
URL         : http://www.hdfgroup.org/HDF5/
Repository  : epel
Summary     : HDF5 development files
Source      : hdf5-1.8.12-13.el7.src.rpm
Description :
HDF5 development headers and libraries.
```

**Comment by stewartsmith** [comment]
Yeah, it's a package from EPEL, not one we have shipped as part of Amazon Linux.

**Comment by dejlek** [comment]
Frankly, without EPEL Amazon Linux 2 would be nearly useless... I doubt people would want to migrate to AL 2023 without something similar to EPEL.

**Comment by daniejstriata** [comment]
I see that the Fedora/EPEL builds are much older than what is current. The recommended version of is 1.14.3.
https://portal.hdfgroup.org/downloads/index.html

Fedora SRC RPM:
https://src.fedoraproject.org/rpms/hdf5

```
Fedora 	hdf5-1.12.1
EPEL 9	hdf5-1.12.1
EPEL 8	hdf5-1.10.5
EPEL 7	hdf5-1.8.12
```
I downloaded and extracted the precompiled version for Centos 8 and was able to run them.
https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.14/hdf5-1.14.3/bin/

**Comment by husseinmimiHarri** [comment]
any updates ??

**Comment by husseinmimiHarri** [comment]
https://repost.aws/articles/ARssMZ5BiQQrufNPpflEv3kQ/installing-python-package-tensorflow-on-amazon-linux-2023-for-graviton

a way to get it

**Comment by dejlek** [comment]
Yes, I started building own set of packages for AL2023 using Fedora's SRPM files...

**Comment by SreeSunku** [comment]
Thank you for your patience. This package is now available as part of the SPAL Repository for AL2023. Please use these resources to familiarize yourself on installing SPAL repository and the packages.
 [SPAL Technical Documentation](https://docs.aws.amazon.com/linux/al2023/ug/configure-spal-repository.html), [SPAL list of packages](https://docs.aws.amazon.com/linux/al2023/release-notes/al-2023-spal-packages.html)
